api/helpers/connection.go

```diff
@@ -0,0 +1,35 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// ConnectDB : This is helper function to connect mongoDB
+func ConnectDB() *mongo.Collection {
+
```

Comment:
i am not completely sure but i guess context works on func to func basis so keeping a parent context and using that here will help us. also ping wala we need to create another func with context background.
let me know if you find anything else